packages/components/src/mobile/progress-circle/index.native.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import Svg, { Circle } from 'react-native-svg';
+import Animated from 'react-native-reanimated';
+
+/**
+ * Internal dependencies
+ */
+
+const ProgressCircle = ( { progress, width = 20, strokeWidth = 2 } ) => {
+	const circleSize = width;
+	const r = ( circleSize - strokeWidth ) / 2;
+	const cx = circleSize / 2;
+	const cy = circleSize / 2;
+
+	const { interpolate, multiply } = Animated;
+	const { PI } = Math;
+	const circumference = r * 2 * PI;
+	const α = interpolate( progress, {
```

Comment:
Hah, interesting constant name :) 
Maybe we should continue using latin letter 🤔